.github/workflows/gradle-wrapper-validation.yml

```
@@ -15,4 +15,4 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: gradle/wrapper-validation-action@v3.3.2
+      - uses: gradle/wrapper-validation-action@v3.4.1
```

```suggestion
      - uses: gradle/actions/wrapper-validation@v3.4.1
```

See https://github.com/gradle/wrapper-validation-action/releases/tag/v3.4.2